src/deroff.rs

```diff
@@ -383,15 +400,40 @@ impl Deroffer {
             .next()
             .map(|(idx, charr)| &string[idx..(idx + charr.len_utf8())]) // Okay to directly index based on idx/charr construction.
             .unwrap_or_default()
+    } */
+
+
+    fn is_white(&self, idx: usize) -> bool {
+        self.s               // String
+            .chars()         // Chars
+            .nth(idx)        // Option<char>
+            .unwrap_or('a')  // char
+            .is_whitespace() // bool
     }
 
-    fn is_white<'a>(s: &'a str, idx: usize) -> bool {
-        match Self::str_at(s, idx) {
-            "" => false,
-            c => c.chars().all(|c| c.is_whitespace()),
-        }
+    /* fn is_white<'a>(s: &'a str, idx: usize) -> bool {
+        s
+          .chars()  // Chars
+          .nth(idx) // Option<char>
+          .unwrap_or('a') // char
+          .is_whitespace() // bool
+    } */
+
+    // This is also known as `prch` apparently
+    fn not_whitespace(&self, idx: usize) -> bool {
```

Comment:
Why not rename this back to `prch`, the same as the original version?

Reply:
I implemented it separately in c40304f just to make sure my implementation didn't break anything, after some testing, my implementation was functionally equivalent to `not_whitespace`. I don't know if there's a reason it was renamed, (cc @scooter-dangle) but the only reason I didn't merge the functions in #68 is because its an associated function rather than a method, and this was the PR for fixing all of those

Reply:
I think @PureW had brought it up (?) when we were looking at `prch` that the function name made no sense—especially when compared to some of the other functions like `is_white` and `digit`. We generally don't rename things as part of the initial port, but this one seemed especially egregious, so we made an exception. Helpful that you added the `prch` comment, @charliethomson, in case someone is looking for what happened to that function.